cli.js

```diff
@@ -69,6 +71,7 @@ try {
 
 function makePromise() {
   let accept, reject;
+  // eslint-disable-next-line promise/param-names
```

Comment:
We can probably fix this. In general I don't think we should ever inline-silence lints. If they can't be fixed then we shouldn't have that rule in the lint rules at all.

Reply:
I've disabled the rule entirely, I think webaverse code tends to use `accept` instead of `resolve` anyway?

Reply:
I'm fine moving to `resolve` -- we did that elsewhere. That would be fine to do here as well.

The argument is we shouldn't be adding junk inline code to please a linter -- we should either fix it or not use that rule.